list/list.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/list/list.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="example-container">
+      <h3 class="h3">Unordered lists</h3>
+      <div class="list-container">
+        <ul class="text-size-md">
+          <li>List item-1</li>
```

Comment:
The ul tags have some user agent style sheet styling so try to reset it

Reply:
This is just a preview for just the components as of now, will work on styling these after I merge with the home page in dev. Will take care of the margin and user-agent stylesheet.